src/frontend/src/common/components/library/data_subview/components/EditSamplesConfirmationModal/index.tsx

```diff
@@ -12,6 +14,13 @@ import {
   InstructionsNotSemiBold,
   InstructionsSemiBold,
 } from "src/components/TreeNameInput/style";
+import { WebformTable } from "src/components/WebformTable";
+import { SAMPLE_EDIT_WEBFORM_METADATA_KEYS_TO_HEADERS } from "src/components/WebformTable/common/constants";
+import {
+  SampleEditMetadataWebform,
+  SampleIdToEditMetadataWebform,
+} from "src/components/WebformTable/common/types";
+import { ContinueButton } from "src/views/Upload/components/common/style";
 import { NewTabLink } from "../../../NewTabLink";
```

Comment:
nit: not related to your PR at all, but this probably should be an absolute import, since these components aren't tied together in any way

Reply:
np, i can fix it while i'm here